meson.build

```diff
@@ -0,0 +1,56 @@
+project(
+    'python-rtmidi',
+    'cpp',
+    version: '1.5.0rc1',
+    license: 'MIT',
+    default_options: [
+        'warning_level=2'
+    ],
+    meson_version: '>=0.53.0'
+)
+
+# Dependencies
+cpp = meson.get_compiler('cpp')
+jack2_dep = dependency('jack', version:'>=1.9.11', required: false)
+jack1_dep = dependency('jack', version:'>=0.125.0, <1.0', required: false)
+alsa_dep = dependency('alsa', required: false)
+thread_dep = dependency('threads')
+coremidi_dep = dependency('threads', required: false)
+winmm_dep = dependency('threads', required: false)
+
+if not jack2_dep.found() and jack1_dep.found()
+    jack_dep = jack1_dep
+elif jack2_dep.found()
+    jack_dep = jack2_dep
+else
+    warning('No version of JACK found, which is recent enough (jack2>=1.9.11 or jack1>=0.125.0)')
+endif
+
+pymod = import('python')
+python = pymod.find_installation(
+    get_option('python'),
+    required: true
+)
+
+# Installation directories
+prefix = get_option('prefix')
+bindir = join_paths(prefix, get_option('bindir'))
+
+# Build jack_mix_box and generate _jack_mixer extension source
+subdir('src')
+
+# Build & install C extension module and Python package
+subdir('rtmidi')
+
+if not get_option('wheel')
+    meson.add_install_script('meson_postinstall.py')
```

Comment:
You should run this with the same python version you found in `pymod.find_installation()` as that may be a different python version than the one in $PATH as `python3`. So you want to make sure that it compiles the right format of bytecode.

Reply:
Fixed in dddbaeb80b5147187841caf6aa0ca395bbe520b5.